### DISCUSSION TITLE
`--verbose` option does not work with `php artisan test` in Laravel 10

### DISCUSSION THREAD

**Discussion by hikaen2** [laravel/framework#46672] [Q&A] [answered]
I got an `Unknown option "--verbose"` error when running `php artisan test --verbose` in Laravel 10.

Reproduce:

```
$ composer create-project laravel/laravel:10.0.5 example-app
$ cd example-app
$ php artisan test --verbose
PHPUnit 10.0.19 by Sebastian Bergmann and contributors.

Unknown option "--verbose"
```

on the other hand, `--help` lists the `--verbose` option:

```
$ php artisan test --help
Description:
  Run the application tests

Usage:
  test [options]

Options:
      --without-tty         Disable output to TTY
      --compact             Indicates whether the compact printer should be used
      --coverage            Indicates whether code coverage information should be collected
      --min[=MIN]           Indicates the minimum threshold enforcement for code coverage
  -p, --parallel            Indicates if the tests should run in parallel
      --profile             Lists top 10 slowest tests
      --recreate-databases  Indicates if the test databases should be re-created
      --drop-databases      Indicates if the test databases should be dropped
      --without-databases   Indicates if database configuration should be performed
  -h, --help                Display help for the given command. When no command is given display help for the list command
  -q, --quiet               Do not output any message
  -V, --version             Display this application version
      --ansi|--no-ansi      Force (or disable --no-ansi) ANSI output
  -n, --no-interaction      Do not ask any interactive question
      --env[=ENV]           The environment the command should run under
  -v|vv|vvv, --verbose      Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

![image](https://user-images.githubusercontent.com/4222502/229565022-1f558dcb-58db-4126-a781-f078e1bf0248.png)

Where is the problem?

**Comment by crynobone** [comment, marked as answer]
https://github.com/sebastianbergmann/phpunit/blob/main/ChangeLog-10.0.md#1000---2023-02-03

PHPUnit 10 removes `--verbose` option but it is a default options provided by Symfony Console

**Reply by blauweb** [reply]
Then how to trigger a verbose logging when running tests? Because by default, phpunit displays only the location of the error.

**Reply by antimech** [reply]
@blauweb `php artisan test --log-events-verbose-text test.log`

Fixed link: https://github.com/sebastianbergmann/phpunit/blob/10.0.0/ChangeLog-10.0.md

**Reply by Braunson** [reply]
Note using `--log-events-verbose-text` doesn't work when using `--parallel`

**Reply by kirkbushell** [reply]
Gawd, why would they remove verbose as an option?! I was wondering why the hell the option wasn't working...

**Reply by sbaum2s** [reply]
I was trying to figure out why `--filter` does not find a specific test. I dont find any working solution. To find out whether the path is wrong, let phpunit talk more to me. `--verbose` options looked good, now it turns out it doesnt exist anymore. Is there a way for extensive logging? Because `--log-events-verbose-text` doesnt really tell me more.
